gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -25,22 +32,37 @@ def ask_questions
       ask_for_unique_cluster_node_number
       ask_for_database_credentials
       ask_for_standby_host
+      ask_for_repmgrd_configuration
       return false if repmgr_configured? && !confirm_reconfiguration
-      confirm(:including_standby_host)
+      confirm
+    end
+
+    def confirm
+      super
+      say(<<-EOS)
+        Standby Host:               #{standby_host}
+        Automatic Failover:         #{run_repmgrd_configuration ? "enabled" : "disabled"}
+      EOS
+      agree("Apply this Replication Server Configuration? (Y/N): ")
     end
 
     def ask_for_standby_host
       self.standby_host = ask_for_ip_or_hostname("Standby Server hostname or IP address", standby_host)
     end
 
+    def ask_for_repmgrd_configuration
+      self.run_repmgrd_configuration = ask_yn?("Configure repmgrd for automatic failover")
```

Comment:
Should we make this a bit more human-readable with `"Configure Replication Manager (repmgrd) for automatic failover"`